components/base/Navbar.vue

```diff
@@ -29,161 +29,161 @@
-                <a
-                  href="/services/consulting/advanced-research-computing"
-                  tabindex="0"
-                >
-                  Consulting
-                </a>
-              </li>
-            </ul>
-          </li>
-          <li tabindex="0" class="navbar-item has-dropdown is-hoverable">
-            <nuxt-link
-              class="navbar-link"
-              :to="{
-                name: 'main',
-                params: {
-                  main: 'our-work'
-                }
-              }"
-            >
-              Our Work
-            </nuxt-link>
-            <ul class="navbar-dropdown">
-              <li class="navbar-item">
-                <a href="/our-work/workshops" tabindex="0">
-                  Workshops
-                </a>
-              </li>
-              <li class="navbar-item">
-                <a href="/our-work/software" tabindex="0">
-                  Software
-                </a>
-              </li>
-              <li class="navbar-item">
-                <a href="https://publications.ccv.brown.edu" tabindex="0">
-                  Publications
-                </a>
-              </li>
-              <li class="navbar-item">
-                <a href="/our-work/talks" tabindex="0">
-                  Talks and Conferences
-                </a>
-              </li>
-            </ul>
-          </li>
-          <li tabindex="0" class="navbar-item has-dropdown is-hoverable">
-            <nuxt-link
-              class="navbar-link"
-              :to="{
-                name: 'main',
-                params: {
-                  main: 'help'
-                }
-              }"
-            >
-              Help
-            </nuxt-link>
-            <ul class="navbar-dropdown">
-              <li class="navbar-item">
-                <a href="mailto:support@ccv.brown.edu" tabindex="0">
-                  Submit a Ticket
-                </a>
-              </li>
-              <li class="navbar-item">
-                <a href="https://docs.ccv.brown.edu" tabindex="0">
-                  Documentation
-                </a>
-              </li>
-              <li class="navbar-item">
-                <nuxt-link
-                  :to="{
-                    name: 'main',
-                    params: {
-                      main: 'help'
-                    }
-                  }"
-                >
-                  More Help Options
-                </nuxt-link>
-              </li>
-            </ul>
-          </li>
-          <li class="navbar-item">
-            <nuxt-link
-              :to="{
-                name: 'main',
-                params: {
-                  main: 'about'
-                }
-              }"
-            >
-              About
-            </nuxt-link>
-          </li>
-        </template>
+        <li class="navbar-item has-dropdown is-hoverable">
+          <nuxt-link
+            class="navbar-link"
+            :to="{
```

Comment:
what is the difference to having `to: /services`? These are the slow ones, the sub--menus, are specified as `<nuxt-link to:"/services/classroom>` for instance, instead of passing to `main`